docs/hugo/content/introduction/annotations.md

```diff
@@ -0,0 +1,27 @@
+# Annotations understood by the operator
+
+## Annotations specified by the user
+
+Note that unless otherwise specified, allowed values are _case sensitive_ and should be provided in lower case.
+
+1. `serviceoperator.azure.com/reconcile-policy`: Specifies the reconcile policy to use. Allowed values are:
+    - `run`: The operator performs all actions as normal. This is the default if no annotation is specified.
```

Comment:
I'm not sure that `run` is an obivous term that someone would understand without reference to documentation. 

How about `fully-managed`?

Reply:
is `fully` adding a lot here? Maybe just `managed`?

Reply:
I changed this to `manage`. Let me know what you think.

Reply:
Works for me.